components/org.wso2.carbon.identity.user.endpoint/src/main/java/org/wso2/carbon/identity/user/endpoint/impl/MeApiServiceImpl.java

```diff
@@ -26,18 +26,23 @@
 import org.wso2.carbon.identity.recovery.IdentityRecoveryClientException;
 import org.wso2.carbon.identity.recovery.IdentityRecoveryConstants;
 import org.wso2.carbon.identity.recovery.IdentityRecoveryException;
+import org.wso2.carbon.identity.recovery.RecoveryScenarios;
+import org.wso2.carbon.identity.recovery.RecoverySteps;
 import org.wso2.carbon.identity.recovery.bean.NotificationResponseBean;
+import org.wso2.carbon.identity.recovery.confirmation.ResendConfirmationManager;
+import org.wso2.carbon.identity.recovery.model.UserRecoveryData;
 import org.wso2.carbon.identity.recovery.signup.UserSelfRegistrationManager;
 import org.wso2.carbon.identity.user.endpoint.Constants;
 import org.wso2.carbon.identity.user.endpoint.MeApiService;
-import org.wso2.carbon.identity.user.endpoint.dto.SelfUserRegistrationRequestDTO;
-import org.wso2.carbon.identity.user.endpoint.dto.SuccessfulUserCreationDTO;
-import org.wso2.carbon.identity.user.endpoint.dto.SuccessfulUserCreationExternalResponseDTO;
+import org.wso2.carbon.identity.user.endpoint.dto.*;
```

Comment:
Avoid using wildcard imports

Reply:
Addressed in https://github.com/wso2-extensions/identity-governance/pull/414/commits/b5f0c45e2ce7cb40a24f7a051acc52455755cce4